### PR TITLE
add cards

### PR DESCRIPTION
**Pull request by shivamSspirit**

### CODE REVIEW

**Review by zabihshaik — APPROVED** [review]
Everything looks good.

File: Components/clCards/card.css [hunk 1]
```diff
@@ -0,0 +1,637 @@
...
+	font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+	max-width: 18rem;
+	height: 21rem;
+	position: relative;
+	display: block;
+	background: linear-gradient(rgb(241, 179, 210), hotpink);
+	border-bottom-left-radius: 40px;
+	border-bottom-right-radius: 40px;
+	border-top-left-radius: 40px;
+	border-top-right-radius: 40px;
```
Comment (yashpatel1797) [inline Components/clCards/card.css:57]: use shorthand properties.

File: Components/clCards/card.css [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+	font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+	max-width: 30rem;
+	max-height: 10rem;
+	position: relative;
+	display: block;
+	background: linear-gradient(rgb(241, 179, 210), hotpink);
+	border-bottom-left-radius: 40px;
+	border-bottom-right-radius: 40px;
+	border-top-left-radius: 40px;
+	border-top-right-radius: 40px;
```
Comment (yashpatel1797) [inline Components/clCards/card.css:122]: here also

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge. try to use shorthand properties. give proper spacing to code.

File: Components/clCards/card.css [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	justify-items: center;
+	padding: 1.5rem;
+
+}
+
+.default-card{
```
Comment (codewithrishu) [inline Components/clCards/card.css]: needed?

File: Components/clCards/card.css [hunk 4]
```diff
@@ -0,0 +1,637 @@
...
+	font-size: large;
+	font-weight: 600;
+	color: rgb(163, 226, 228);
+}
+
+.info {
+	font-size: medium;
+	font-weight: 400;
+	color: rgb(207, 85, 105);
+	font-weight: 600;
```
Comment (codewithrishu) [inline Components/clCards/card.css:98]: two font weight are present

File: Components/clCards/card.css [hunk 5]
```diff
@@ -0,0 +1,637 @@
...
+	.container {
+		display: block;
+		width: 100%;
+	}
+}
+
+@media screen and (min-width: 900px) {
+	.container {
+		/* width: 33.33333%; */
+	}
```
Comment (codewithrishu) [inline Components/clCards/card.css]: unwanted  code

File: Components/clCards/card.css [hunk 6]
```diff
@@ -0,0 +1,637 @@
...
+}
+
+.container .title {
+	color: #1a1a1a;
```
Comment (codewithrishu) [inline Components/clCards/card.css:179]: don't use colour directly..use variable

**Review by codewithrishu — APPROVED** [review]

**Pull request merged**